Watch Extension/HomeView.swift

```diff
@@ -20,10 +18,11 @@ struct HomeView: View {
     var body: some View {
         // TODO: think more about the layout of these two
         VStack {
-            List(store.value.activeFeedings, id: \.self) { _ in
+            List(store.value.activeFeedings) { feeding in
                 // TODO: setup timer for real
-                Text("00:00:00")
+                FeedingView(store: self.store, feeding: feeding)
             }
+            
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/9397ad90d7ca569afad13609babe3778b69f0d95/Watch Extension/HomeView.swift#L25"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>HomeView.swift:25</code></td>
    </tr>
  </tbody>
</table>